docs/concepts/executor/docarray-v2.md

```diff
@@ -0,0 +1,125 @@
+(docarray-v2)=
+
+
+# (Beta) DocArray v2
+
+Jina provides early support for [DocArray v2](https://github.com/docarray/docarray/commits/feat-rewrite-v2) which
+is a rewrite of DocArray. DocArray v2 makes the dataclass feature of DocArray-v1 a first-class citizen and for this 
+purpose it is built on top of [pydantic](https://pydantic-docs.helpmanual.io/) and . An important shift is that 
```

Comment:
and...what?

Reply:
Or was this an editing glitch on my part from earlier?

Reply:
euh yeah I think so ?

Reply:
can u make i better them ?

Reply:
okay actually it is on the next line nothing wrong

Reply:
Uh, I still see the dangling `and .`

![image](https://user-images.githubusercontent.com/4182659/216587433-dc2b8467-dc4c-4d17-8f38-ad8e94f110e7.png)